cmd/goose/sql.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/go-sql-driver/mysql"
+)
+
+// normalizeMySQLDSN parses the dsn used with the mysql driver to always have
+// the parameter `parseTime` set to true. This allows internal goose logic
+// to assume that DATETIME/DATE/TIMESTAMP can be scanned into the time.Time
+// type.
+func normalizeMySQLDSN(dsn string) (string, error) {
+	config, err := mysql.ParseDSN(dsn)
+	if err != nil {
+		return "", err
+	}
+	config.ParseTime = true
+	return config.FormatDSN(), nil
+}
+
+func createDBWithDriver(driver string, dbstring string) (*sql.DB, error) {
```

Comment:
Can you move this to goose pkg itself, and use this function in both `cmd/goose/main.go` and `examples/go-migrations/main.go` files?

I'd like to keep `cmd/` minimal or even simplify it over time, per requests like #98.